tests/unit/multi-currency/test-class-currency.php

```diff
@@ -16,20 +18,26 @@ class WCPay_Multi_Currency_Currency_Tests extends WP_UnitTestCase {
 	 */
 	private $currency;
 
+	/**
+	 * @var int
+	 */
+	private $timestamp_for_testing;
+
 	/**
 	 * Pre-test setup
 	 */
 	public function setUp() {
 		parent::setUp();
 
-		$this->currency = new WCPay\MultiCurrency\Currency( 'USD' );
+		$this->timestamp_for_testing = strtotime( '1 January 2021' );
+		$this->currency              = new Currency( 'USD', 1.0, $this->timestamp_for_testing );
```

Comment:
Should all of the tests be updated to create new currencies the same? Meaning `new Currency` vs `new WCPay\MultiCurrency\Currency`.

Reply:
Good catch! Fixed in 1724120c36db7a502d0e34e1b08baafb3405c57b.